### ISSUE TITLE
[Request] Config to disable HUD & Particles

### ISSUE THREAD

**Issue by Xaikii** [TheIllusiveC4/Champions#36]
I am working on a Pack where I boost the number of Champions and force Champion Status on some mobs, I noticed the HUD is pretty much always active. I'd wish to have a config option to disable the HUD(including the Naming Feature) such as the Particles. Some Mobs can turn invis and the Particles make them visible, which pretty much destroys their Kit.

**Comment by TheIllusiveC4** [comment]
For clarity, did you envision this as a client-side configuration option or server-side (so that players cannot re-enable them)?

**Comment by Xaikii** [comment]
sry Github for some reasons the Github Mail ended up in Spam and I didnt get notified about your message. 
Yes I envisioned it as client-sided. I noticed you've already implemented it and I checked it out, 
is it intended that the champion effect will only be hidden AFTER the config was set true? 
To clarify, Mobs spawned with Champion Effects HUD BEFORE the config is set to true will keep their Champion Effect HUD even after it is set to true.
